m4/pbs_decl_socklen_t.m4

```diff
@@ -45,9 +45,10 @@ AC_DEFUN([PBS_AC_DECL_SOCKLEN_T],
 #include <netdb.h>
 ], [
 #ifdef  __hpux
-  NO socklen_t in HPUX
+#error NO socklen_t in HPUX
```

Comment:
It's a C pre-processor directive to halt compilation immediately with the provided error text. In this case, we wanted the compilation to fail on HP-UX machines so that socklen_t gets defined as an int. The compilation would have failed anyway due to a syntax error in this case. There's no change in behavior here, just cleanup.